packages/deployments/contracts/src/cli/helpers.ts

```diff
@@ -0,0 +1,55 @@
+import { Contract, Wallet } from "ethers";
+
+import _Deployments from "../../deployments.json";
+import { ConnextInterface } from "../../../contracts";
+
+import { Deployment } from "./types";
+
+const Deployments = _Deployments as any;
+
+// Custom function to format lookup by env and double check that the contract retrieved is not null.
```

Comment:
Hmmm, maybe would be helpful to share `helpers` folder between `init` and `watcher`? i.e. have a shared folder... idk, that might add some complexity with managing any overlap that comes with that, but I think it would be good to curtail any redundancy in this new `cli` group of scripts and stuff

Reply:
i have some shared files

Reply:
i moved some types / common helpers into there